### ISSUE TITLE
Have the switchboard perform automatic conversion from PDF/RTF/DOCX to text/plain

### ISSUE THREAD

**Issue by claus-zinn** [clarin-eric/switchboard#19]
This is related to issue clarin-eric/LRSwitchboard#17 but maybe it is more subtle or implicit.

Tools such as WebLicht are capable of converting from PDF/DOCX to plain text before starting their tool chains. Other tools are only capable of processing plain text. The enhancement proposed: When users upload "rich text" such as PDF, RTF, or Word documents, have the Switchboard *automatically* convert such text to plain text, *before* identifying applicable tools. Note that this increases the number of applicable tools, and it also increases the recognition rate for language detection (as Apache Tika works better on plain text than, say, on PDF documents. The enhancement also centralises conversion at the switchboard. Motto: do it well centrally, rather than doing it with varying quality at the tool providers' site.

**Comment by proycon** [comment]
Though it's a nice idea, I don't really think the switchboard itself should go there (I think the best it can do is call an external service for such conversion, using its normal matching mechanism but that would require some extra facilities for getting the output back again). Some conversions (from PDF or TEI for example) are far from trivial. I implemented (non-automatic) conversions in CLAM too but in practice they are rarely used and in hindsight it was probably not the best place to implement them.

I did develop a separate a data conversion service for CLARIAH called [Piereling](https://github.com/proycon/piereling), which I'd like to add to the switchboard registry (will do a PR soon). It may be a bit too FoLiA-centric to be generally applicable though, I didn't bother to implement all of pandoc's conversions. There's also the INT's [OpenConvert](https://github.com/INL/OpenConvert), which is more TEI-centric.
